<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conversion</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>261</width>
      <height>290</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="DatabaseNameLabel">
       <property name="text">
        <string>Database</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="DatabaseNameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passwordLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="hostsLabel">
       <property name="text">
        <string>hosts</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="hostsLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="userLabel">
       <property name="text">
        <string>user</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="userLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="mongoClientNameLabel">
       <property name="text">
        <string>Mongo database</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="mongoClientNameLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="collectionNameLabel">
       <property name="text">
        <string>Collection name</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="collectionNameLineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="sQLTabelNameLabel">
       <property name="text">
        <string>SQL Tabel Name</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="sQLTabelNameLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Submit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
